docs/DeveloperGuide.md

```diff
@@ -1,38 +1,212 @@
+## Acknowledgements
+1. Reference to AB-3 Developer Guide: https://se-education.org/addressbook-level3/DeveloperGuide.html
+2. Reference to AB-3 diagrams code: https://github.com/se-edu/addressbook-level3/tree/master/docs/diagrams
+
+## Setting up, getting started
+### Setting up the project in your computer
+Firstly, **fork** this repo, and **clone** the fork into your computer. <br>
+<br>
+If you plan to use Intellij IDEA (highly recommended): <br>
+1. **Configure the JDK**: Follow the guide 
+[[se-edu/guides] IDEA: Configuring the JDK](https://se-education.org/guides/tutorials/intellijJdk.html) 
+to ensure Intellij is configured to use **JDK 11**.
+2. **Import the project as a Gradle project**: Follow the guide 
+[[se-edu/guides] IDEA: Importing a Gradle project](https://se-education.org/guides/tutorials/intellijImportGradleProject.html) 
+to import the project into IDEA.<br>
+   **Note**: Importing a Gradle project is slightly different from importing a normal Java project.
+3. **Verify the setup:**
+   1. Run the ```wellnus.WellNus``` and try a few commands.
+   2. Run the tests to ensure they all pass.
+
+### Before writing code
+1. **Configure the coding style**<br>
+If using IDEA, follow the guide 
+[[se-edu/guides] IDEA: Configuring the code style](https://se-education.org/guides/tutorials/intellijCodeStyle.html) 
+to set up IDEA’s coding style to match ours.<br>
+<br>
+2. **Set up CI**<br>
+   This project comes with a GitHub Actions config files (in `.github/workflows` folder). 
+When GitHub detects those files, it will run the CI for your project automatically at each push 
+to the `master` branch or to any PR. No set up required.<br>
+<br>
+3. **Learn the design**<br>
+When you are ready to start coding, we recommend that you look at the class diagrams to understand the structure of the 
+code and the interaction among different classes.<br>
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Object Diagram
 
 ## Product scope
 ### Target user profile
-
-{Describe the target user profile}
+* NUS Computing and Engineering students
+* Spend lots of time coding on their IDE and type relatively fast
+* Have to regularly use digital gadgets and Internet for their courses
+* Very familiar with command line interfaces
+* Stressed about academy and many others
+* Busy with work and drowning in deadlines
+* Wants to improve their wellness
+* Sometimes unmotivated with short attention span
 
 ### Value proposition
-
-{Describe the value proposition: what problem does it solve?}
+NUS Computing and Engineering students are often busy with work and sometimes will neglect their wellness. This app aims
+to help NUS Computing and Engineering students improve their overall wellness by encouraging the **cultivation of meaningful
+atomic habits**, **practice of self reflection** and **usage of offline timer to stay focused**. By using this app, 
```

Comment:
Is the **cultivation of meaningful atomic habits** supposed to be bolded as well?

Reply:
Weird GitHub doesn't render the bolded effect, but it renders fine in IntelliJ?